### PR TITLE
Feat/yo basic project

### PR DESCRIPTION
**Pull request by roxhens**
@edgarmueller, I was not able to edit the reviewers, so I am tagging you here.

### CODE REVIEW

File: generator-jsonforms/src/generators/app/index.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Test cases for the Generators package.
+ */
+describe('Generators package tests', () => {
```
Comment (edgarmueller) [inline generator-jsonforms/src/generators/app/index.test.js]: Any plans on actually testing something here (applies to other tests as well)?

File: generator-jsonforms/package.json [hunk 2]
```diff
@@ -17,9 +17,11 @@
   },
   "dependencies": {
     "@types/yeoman-generator": "^3.0.1",
+    "@jsonforms/core": "^2.0.12",
     "chalk": "^2.4.2",
     "clear": "^0.1.0",
     "figlet": "^1.2.1",
+    "jsonforms-basic-project": "git+https://git@github.com/roxhens/jsonforms-yo-basic-project.git",
```
Comment (edgarmueller) [inline generator-jsonforms/package.json]: The referenced repo needs to be moved to our organization. But on a different note: is this the same template you use for bootstraping with OpenAPI? There are commits on the referenced project which remove the submit button and so on. There should be two templates one for the 'basic' use case and another for the OpenAPI one, but it seems there's only tempalte proejct. Or am I missing something here?

File: generator-jsonforms/README.md [hunk 3]
```diff
@@ -42,6 +44,17 @@ Command:
...
 
+### Schema Path
+
+Enter the URL where the JSON schema is located
+
+This is important for the Basic project to work.
+
+Command:
+```shell
+yo jsonforms --project "basic" --basicProjectSchemaPath "/Users/roxhens/Desktop/tester/project_files/schema_file.json"
```
Comment (edgarmueller) [inline generator-jsonforms/README.md]: Make the example path more generic

File: generator-jsonforms/package.json (hunk 2 above)
Comment (roxhens) [reply generator-jsonforms/package.json]: I updated the url with an eclipsesource repo. Concerning the commits that remove the submit button, they are not present in the new repo.

File: generator-jsonforms/src/generators/app/index.test.js (hunk 1 above)
Comment (roxhens) [reply generator-jsonforms/src/generators/app/index.test.js]: Until the final presentation we just set up the tests and could not complete them.

**Pull request closed without merging**